lfs/gitscanner_catfilebatchcheckscanner_test.go

```diff
@@ -0,0 +1,34 @@
+package lfs
+
+import (
+	"bufio"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCatFileBatchCheckScanner(t *testing.T) {
```

Comment:
Hm, I think this test name could be more specific: `TestCatFileBatchCheckScansCorrectlyFormattedBlobs`.